README.md

```diff
@@ -320,20 +320,20 @@ Games using this method include [Coins & Steel](https://opensea.io/assets/coins&
 
 ### Fetching Orders
 
-To retrieve a list of offers and auction on an asset, you can use an instance of the `OpenSeaAPI` exposed on the client. Parameters passed into API filter objects are underscored instead of camel-cased, similar to the main [OpenSea API parameters](https://docs.opensea.io/v1.0/reference):
+To retrieve a list of offers and auction on an asset, you can use an instance of the `OpenSeaAPI` exposed on the client. Parameters passed into API filter objects are camel-cased and serialized before being sent as [OpenSea API parameters](https://docs.opensea.io/v1.0/reference):
```

Comment:
really minor - change to 2.0 docs: ```https://docs.opensea.io/v1.0/reference```

Reply:
Thanks 🙏  will include here
https://github.com/ProjectOpenSea/opensea-js/pull/649